test/kb_quast_server_test.py

```diff
@@ -317,7 +314,7 @@ def test_fail_bad_ws_type(self):
             {'assemblies': [bad_object_ref]},
             "Cannot write data to fasta; invalid WS type ({}).  ".format(bo_type) +
             "Supported types are KBaseGenomes.ContigSet and KBaseGenomeAnnotations.Assembly",
-            exception=AssemblyError)
+            exception=AssemblyError, contains=True)
```

Comment:
What changed such that contains is necessary?

Reply:
See above